clang/test/CIR/CodeGen/complex.cpp

```diff
@@ -1495,3 +1495,42 @@ void calling_function_that_return_complex() {
 // OGCG: %[[A_IMAG_PTR:.*]] = getelementptr inbounds nuw { float, float }, ptr %[[A_ADDR]], i32 0, i32 1
 // OGCG: store float %[[RESULT_REAL]], ptr %[[A_REAL_PTR]], align 4
 // OGCG: store float %[[RESULT_IMAG]], ptr %[[A_IMAG_PTR]], align 4
+
+void imag_literal_with_gun_extension() {
```

Comment:
I am guessing this is supposed to say `imag_literal_gnu_extension`?
The commit message should mention "literal" in the subject line: Add test for GNU i literal suffix for Complex imag

@AaronBallman, what is your preference on correcting this (including the commit message)? Revert and land fixed version?